lib/web.js

```diff
@@ -87,7 +85,9 @@ var SolidWebClient = {
     return new Promise(function (resolve, reject) {
       var http = new XMLHttpRequest()
       http.open(method, url)
-      http.withCredentials = true
+      if (!options.noCredentials) {
+        http.withCredentials = true
```

Comment:
By default, the `options` hash does not have `noCredentials` defined. So, by default, credentials are set. Is that what you're asking?